selector/similarity.py

```diff
@@ -25,7 +25,7 @@
 
 import numpy as np
 
-__all__ = ["pairwise_similarity_bit", "tanimoto", "modified_tanimoto"]
+__all__ = ["pairwise_similarity_bit", "tanimoto", "modified_tanimoto", "scaled_similarity_matrix"]
```

Comment:
Maybe it's preferred to put it as below, so that future changes will appear as a new line

```
__all__ = [
    "pairwise_similarity_bit", 
    "tanimoto", 
    "modified_tanimoto", 
    "scaled_similarity_matrix",
]
```